src/authentication/calls.ts

```diff
@@ -11,18 +11,18 @@ export interface GetTokenInput {
 
 export const authCodeResponseSchema = z
   .object({
-    id_token: z.string().optional(),
-    access_token: z.string(),
-    expires_in: z.number(),
-    token_type: z.literal('Bearer'),
-    refresh_token: z.string().optional(),
+    idToken: z.string().optional(),
+    accessToken: z.string(),
+    expiresIn: z.number(),
+    tokenType: z.literal('Bearer'),
+    refreshToken: z.string().optional(),
```

Comment:
This won't work. This schema is used to parse the reply from the API, where the fields are in snake case. In order to get camel cased data out of this, we need to use `.transform()`. See example below.